website/common/locales/en/groups.json

```diff
@@ -223,11 +223,11 @@
   "inviteMissingUuid": "Missing user id in invite",
   "inviteMustNotBeEmpty": "Invite must not be empty.",
   "partyMustbePrivate": "Parties must be private",
-  "userAlreadyInGroup": "User already in that group.",
+  "userAlreadyInGroup": "User \"<%= username %>\" already in that group. \"<%= userId %>\"",
   "cannotInviteSelfToGroup": "You cannot invite yourself to a group.",
-  "userAlreadyInvitedToGroup": "User already invited to that group.",
-  "userAlreadyPendingInvitation": "User already pending invitation.",
-  "userAlreadyInAParty": "User already in a Party.",
+  "userAlreadyInvitedToGroup": "User \"<%= username %>\" already invited to that group.\"<%= userId %>\"",
+  "userAlreadyPendingInvitation": "User \"<%= username %>\" already pending invitation. \"<%= userId %>\"",
+  "userAlreadyInAParty": "User \"<%= username %>\"  already in a party.\"<%= userId %>\"",
```

Comment:
could you remove the extra space in the sentence, and add the user id in this way ` User-ID: {userId}`, that way people know it's the UserID and not a random number